### PR TITLE
feat: add filters clearing

### PR DESCRIPTION
**Pull request by mmcode9407**

### CODE REVIEW

**Review by NerdbordMentor — APPROVED** [review]
## Code Review Report

### Task goal
The task goal is to implement a feature that allows users to clear all selected filters.

### Requirements
1. UI should be implemented as in design.
2. When clicking on `Clear filters` it should clear all selected filters, and show all available job offers.

### Code Changes Review

The code changes in the file `src/components/Filter/FilterBox.tsx` seem to cover the task requirements. 

The `clearFilters` function from the `FiltersContext` is imported and used as an `onClick` event handler for the `Clear filters` button. This should clear all selected filters when the button is clicked.

```diff
+   const { clearFilters } = useFilters();
...
+            <button className=" w-[95] p-2  text-accent-strong text-md-12" onClick={clearFilters}>
+               Clear filters
+            </button>
```

However, without the context of the `useFilters` hook and the `clearFilters` function, it's hard to definitively say if this will show all available job offers as required. Assuming that the `clearFilters` function is correctly implemented to clear all filters and update the job offers accordingly, this code change should meet the task requirements.

As for the UI, the code changes don't seem to affect it, so it should remain as in the design. However, without seeing the actual UI or the design it's supposed to follow, it's impossible to definitively say if it meets the first requirement.

In conclusion, based on the provided code changes and assuming that other parts of the code (like the `useFilters` hook and the `clearFilters` function) are correctly implemented, these changes seem to cover the task requirements.
